webjar/src/main/webjar/pdfviewer-customizations.js

```diff
@@ -0,0 +1,22 @@
+(function() {
+   window.addEventListener('load', function() {
+      setExternalLinkTarget();
+      setViewRights();
+   }, true);
+
+  // PDFVIEWER-2: Open the links inside the PDF documents in a new tab
+  function setExternalLinkTarget() {
+    PDFViewerApplication.preferences.set('externalLinkTarget', 2);
+  }
+  // PDFVIEWER-13: Allow author of the macro to delegate its view right on the PDF document.
+  function setViewRights() {
+    var file = new URLSearchParams(window.location.search).get('file') || '';
```

Comment:
``URLSearchParams`` is not supported in IE11 unfortunately, which we still need to support... :( Moreover, we don't have Closure Compiler on XWiki 11.10 so we can't rely on it to add the polyfill. This we have two options:

* write the code without relying on ``URLSearchParams``
* try to load an external polyfill such as https://search.maven.org/artifact/org.webjars.npm/url-search-params-polyfill/8.1.1/jar